tests/aiohttp/test_graphql_transport_ws.py

```diff
@@ -420,6 +420,11 @@ async def test_subscription_cancellation(aiohttp_client):
         response = await ws.receive_json()
         assert response == CompleteMessage(id="sub2").as_dict()
 
+        # Issue #1731
```

Comment:
Did we really need to reference the issue in our code? 🤔

Reply:
I don't know, In projects that I have worked on it is customary to link tests to issues if they are regression tests verifying that an issue is fixed.  CPython being one of them.

But I'll follow whatever guidelines set by this project, of course :)

Reply:
I'm not opposed to having the regression test link to the GH issue, but I think it should be a separate unit test.

Reply:
You mean, have a separate .py file for regression tests?  Any suggestions?

Reply:
No, just separate `def test_X` functions. You modified an existing test to check for the regression. I was suggesting that they become distinct tests instead.

Reply:
Agreed, that was sloppy.